express/blocks/quick-action/shared.js

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
I'd move this to a `/express/blocks/shared/quick-action.js` to avoid having two different shared module approaches in the project.

Reply:
This is only for temporary. Once the winning experience is identified, will merge the code of shared.js to quickaction.js and have only one file for the block, rather than browser have to download 2 files. Since this isn't shared with any other functionality, I'd like keep this as-is for now.

cc: @ramboz